### PR TITLE
DV360 (Actions) Docs

### PR DESCRIPTION
**Pull request by maryamsharif**
### Proposed changes

Adding Documentation for our new DV360 Actions Destination. 

### Merge timing

Release on Jan 30th - DV360 Actions Public Beta

### CODE REVIEW

File: src/connections/destinations/catalog/actions-display-video-360/index.md [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+   2. **Google Network Id**: `segment`
+   3. Your [Segment Write Key](/docs/connections/find-writekey/). You can retrieve your write key from the Settings tab on the Source.
+   4. **DoubleClick Advertiser ID**
+      - If you use DoubleClick Floodlight for DV360 only, enter `DV360`.
+      - If you use DoubleClick Floodlight for other use cases in addition to DV360, enter the Advertiser ID from your Doubleclick Floodlight account.
+3.  Switch the toggle to enable the destination.
+
+### Enable and configure the DV360 destination
+
+1. From your Segment workspace, go to **Engage → Engage Settings → Destinations→ Add Destination** and then Search **Display and Video 360 (Actions)**.
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-display-video-360/index.md:94]:
```suggestion
1. From your Segment workspace, navigate to **Engage > Engage Settings > Destinations > Add Destination**, then search for **Display and Video 360 (Actions)**.
```

File: src/connections/destinations/catalog/actions-display-video-360/index.md [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+2. Authenticate using OAuth.
+3. Switch the toggle to enable the destination.
+4. Navigate to the **Mappings** tab, click **Add Mapping** and select **Add to Audience**. 
+5. Click **Save** and make sure to enable the mapping. 
+6. On the **Mappings** tab, click **Add Mapping** and select **Remove from Audience**. 
+7. Click **Save** and make sure you enable the mapping. 
+
+> info ""
+> **Note**: The destination does not have configurable settings until you create an audience, described [here](#create-an-audience-and-finish-dv360-configuration)
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-display-video-360/index.md:103]:
```suggestion
> The destination does not have configurable settings until you create an audience, described [here](#create-an-audience-and-finish-dv360-configuration).
```

File: src/connections/destinations/catalog/actions-display-video-360/index.md [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+
+### Create an audience and finish DV360 configuration
+
+Create an [Audience](/docs/personas/audiences) in a new or existing Engage space. After you create the audience, you can select the Display & Video 360 (Actions) destination you created before.
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-display-video-360/index.md:107]:
```suggestion
[Create an audience](/docs/personas/audiences) in a new or existing Engage space. After you create the audience, you can select the Display & Video 360 (Actions) destination you created before.
```

File: src/connections/destinations/catalog/actions-display-video-360/index.md [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+
+> info ""
+> **Note**: These settings are tied to a single audience. Each additional audience you send to DV360 requires you to input these values.
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-display-video-360/index.md:110]:
```suggestion
> These settings are tied to a single audience. Each additional audience you send to DV360 requires you to input these values.
```

File: src/connections/destinations/catalog/actions-display-video-360/index.md [hunk 5]
```diff
@@ -0,0 +1,222 @@
...
+
+When you select the destination, you're prompted to complete the destination settings:
+
+| Setting                        | Description                                                                                                                                                               |
+| ------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Account Type                   | The type of DV360 account used to sync. Either `Advertiser`, `Partner`, or `Publisher`. **Note:** Select `Publisher` only if you plan to connect to Google Ad Manager.    |
+| Advertiser ID                  | The ID of your DV360 Advertiser account. Can be found in your Google Account under Advertiser Settings → Basic Details → Advertiser ID.                  {:target="_blank"}.                                                                                                                     |
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-display-video-360/index.md:117]:
```suggestion
| Advertiser ID                  | The ID of your DV360 Advertiser account. Can be found in your Google Account under **Advertiser Settings > Basic Details > Advertiser ID**.                                                                                                                                       |
```

File: src/connections/destinations/catalog/actions-display-video-360/index.md [hunk 6]
```diff
@@ -0,0 +1,222 @@
...
+
+Also toggle on the Send Track setting. 
+
+> info ""
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-display-video-360/index.md:121]:
```suggestion

```

File: src/connections/destinations/catalog/actions-display-video-360/index.md [hunk 7]
```diff
@@ -0,0 +1,222 @@
...
+Extra information from the API (settings, connection modes, etc.) are automatically pulled in here.
+{% include content/sync-frequency-note.md %}
+
+## Migrate from Personas Google Display & Video 360 Destination 
+
+Segment will copy all of your existing Personas Display & Video 360 Destination configurations to Display and Video 360 (Actions). Once the migration is completed , you will be notified by email. 
+
+The sole mandatory action involves **re-authenticating** for each migrated Personans Display & Video 360 Destination. While you may already possess OAuth credentials, it is essential to undergo reauthentication. This step is necessary to seamlessly integrate new audiences or implement updates to existing ones.
+
+We are disabling all existing Personas Display and Video 360 destinations. You can still access your existing configuration, but please refrain from enabling the destination, as it is set to be deprecated. You will no longer be able to create new instances of Personas Display and Video 360.
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-display-video-360/index.md:134]:
```suggestion
Segment is disabling all existing Personas Display and Video 360 destinations. You can still access your existing configuration, but please refrain from enabling the destination, as it is set to be deprecated. You will no longer be able to create new instances of Personas Display and Video 360.
```

File: src/connections/destinations/catalog/actions-display-video-360/index.md [hunk 8]
```diff
@@ -0,0 +1,222 @@
...
+
+When you complete the connection between Segment and DV360, it can take from 24 to 48 hours for Google to create the user list. This must complete before Segment can begin to sync users into that list.
+
+
+### What identifiers are needed to enable this integration?
+
+Google's [documentation](https://developers.google.com/authorized-buyers/rtb/downloads/cookie-bulk-upload-proto){:target="_blank"} provides information about the accepted identifiers for this integration.
+
+
+- To use DV360 with web traffic, you must collect `anonymous_id` through the client-side `analytics.js` Source.
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-display-video-360/index.md:155]:
```suggestion
- To use DV360 with web traffic, you must collect `anonymous_id` through the client-side `analytics.js` source.
```

File: src/connections/destinations/catalog/actions-display-video-360/index.md [hunk 9]
```diff
@@ -0,0 +1,222 @@
...
+
+
+1. During an audience sync, Segment sends a list of `anon_id` values to Google. Google attempts to match those values in their match table, to find an associated `google_user_id`.
+2. To complete this lookup, Google must have both the `anon_id` and have it store along side a matched `google_user_id`. This occurs when a user visits your website with both the Doubleclick Floodlight tag installed, and the DV360 integration completed.
+
+As a result, you must have Doubleclick Floodlight and the DV360 integration in place before Google can match users and make them available for retargeting.
+
+To help reduce the difference between Engage and DV360 audience sizes, Segment recommends that you deselect the `Historical Backfill` option when you create the audience that syncs to DV360.
+
+#### Impact to third-party cookies: Browser Policies
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-display-video-360/index.md:175]:
```suggestion
#### Impact to third-party cookies: browser policies
```

File: src/connections/destinations/catalog/actions-display-video-360/index.md [hunk 10]
```diff
@@ -0,0 +1,222 @@
...
+
+The DV360 integration for web traffic relies on Doubleclick Floodlight, which itself relies on a `google_user_id` cookie. While this cookie is “yours”, browsers treat this as a third-party cookie because it is served from Google's servers, and not the same domain as your website. As browsers become more privacy-oriented, they block third-party cookies by default.
+
+Users who visit your website in Firefox and Safari, and who do not specifically allow third-party cookies, are not identifiable by Doubleclick Floodlight (`google_user_id`). This prevents Google from identifying a match between an `anon_id` sent from Segment, and results in lower match rates.
+
+#### Impact to third-party cookies: Adblockers
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-display-video-360/index.md:181]:
```suggestion
#### Impact to third-party cookies: adblockers
```

File: src/connections/destinations/catalog/actions-display-video-360/index.md [hunk 11]
```diff
@@ -0,0 +1,222 @@
...
+
+All browser-based adblocking software intentionally blocks most third-party cookies, including the Doubleclick Floodlight cookie necessary for identification. As a result, Google cannot match users who employ adblocking software in their browsers.
+
+#### IDFA impact: Recent Apple announcements
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-display-video-360/index.md:185]:
```suggestion
#### IDFA impact: recent Apple announcements
```

File: src/connections/destinations/catalog/actions-display-video-360/index.md [hunk 12]
```diff
@@ -0,0 +1,222 @@
...
+
+Apple has announced an updated privacy policy that, while not rolled out yet, impacts the way businesses collect IDFA data. When enacted, this privacy policy will significantly reduce the percentage of users for which IDFA data is collected. This change will result in lower match rates, as both Segment and Google will see a decline in the number of IDFA values sent by Segment, and matched by Google.
+
+#### Invalid Google IDs
+
+Sometimes, Google denies IDFA or `google_user_id` values when they consider them to be invalid or inactive.
+
+#### Modifying Lists Configuration in DV360
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-display-video-360/index.md:193]:
```suggestion
#### Modifying list configuration in DV360
```

File: src/connections/destinations/catalog/actions-display-video-360/index.md [hunk 13]
```diff
@@ -0,0 +1,222 @@
...
+
+Engage syncs every IDFA or `anonymous_id` value for each user in an audience. When DV360 receives this data, it does not de-duplicate in the event that multiple identifiers map to the same unique user. This may result in a larger audience list in Google compared to Engage.
+
+
+### Why don't I see matches in DV360?
+
+The most common cause of matches not appearing in DV360 is an error with Doubleclick Floodlight. From the website where tracking is enabled, open the Network inspector, and confirm that outgoing requests to `idsync.segment.com` appear.
+
+
+### How does third-party cookie eradication impact the DV360 Destination?
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-display-video-360/index.md:208]:
```suggestion
### How does third-party cookie eradication impact the DV360 destination?
```

File: src/connections/destinations/catalog/actions-display-video-360/index.md [hunk 14]
```diff
@@ -0,0 +1,222 @@
...
+Google Chrome has committed to replacing third-party cookies with an alternative, but has not announced a timeframe for that alternative. Segment will not update this integration until these updates from Google are announced.
+
+
+### Can I use Engage audiences to target YouTube ads with this integration?
+
+No. YouTube (through DV360) does not support the type of lists that Segment provides.
+
+### Why do I see destination settings after I add my audience, but not when I first enable the destination?
+
+The DV360 Destination works on a per-audience basis. This enables you to:
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-display-video-360/index.md:219]:
```suggestion
The DV360 destination works on a per-audience basis. This enables you to:
```

File: src/connections/destinations/catalog/actions-display-video-360/index.md [hunk 15]
```diff
@@ -0,0 +1,222 @@
...
+---
+
+> info Display and Video 360 (Actions) operates using third-party cookies, its match rates are influenced by the extent to which these cookies are supported by browsers
+
+Google's [Display & Video (DV360)](https://marketingplatform.google.com/about/display-video-360/){:target="_blank"} service is an end-to-end campaign management tool that enables enterprise customers to plan, measure, and run display and video advertisements. 
+
+> info ""
+> **Note**: You can connect to a Google Ad Manager account. For more information, see [4. Create an audience and finish DV360 configuration](#4-create-an-audience-and-finish-dv360-configuration) below. When filling out the `User-Role Granted` selected `Publisher` if you place to connect to Google Ad Manager.
+
+Segment's integration with DV360 enables Segment customers to sync audiences created in Engage with DV360 to enable centralized audience management and improved retargeting.
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-display-video-360/index.md:17]:
```suggestion
Segment's integration with DV360 enables Segment customers to sync audiences created in Engage with DV360 for centralized audience management and improved retargeting.
```

File: src/connections/destinations/catalog/actions-display-video-360/index.md [hunk 16]
```diff
@@ -0,0 +1,222 @@
...
+
+Segment requires the [DoubleClick Floodlight](/docs/connections/destinations/catalog/doubleclick-floodlight/) tag on your website to enable the creation of audiences based on website traffic. This allows Segment to send Google the appropriate identifier (typically `anonymousId`) for users that are in an audience. Google stores these identifiers on its servers and matches them against `google_id`.
+
+To configure DoubleClick Floodlight:
+
+> warning ""
+> **Prerequisite**: Create a [JavaScript Website](/docs/connections/sources/catalog/libraries/website/javascript/) source in your Segment workspace if one does not exist. Ensure that this source is configured to track visitors to your website. For more information about configuring Javascript sources, see the [Analytics.js Quickstart guide](/docs/connections/sources/catalog/libraries/website/javascript/quickstart/).
+
+1. In your workspace, visit the **Catalog** and search for the **DoubleClick Floodlight** destination.
+2. Connect your JavaScript Website source to the DoubleClick Floodlight destination, and configure the following settings:
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-display-video-360/index.md:83]:
```suggestion
2. Connect your JavaScript website source to the DoubleClick Floodlight destination, and configure the following settings:
```

File: src/connections/destinations/catalog/actions-display-video-360/index.md [hunk 17]
```diff
@@ -0,0 +1,222 @@
...
+   1. **Get DoubleClickID**: `On`
+   2. **Google Network Id**: `segment`
+   3. Your [Segment Write Key](/docs/connections/find-writekey/). You can retrieve your write key from the Settings tab on the Source.
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-display-video-360/index.md:86]:
```suggestion
   3. Your [Segment Write Key](/docs/connections/find-writekey/). You can retrieve your write key from the Settings tab in the source.
```

File: src/connections/destinations/catalog/actions-display-video-360/index.md [hunk 18]
```diff
@@ -0,0 +1,222 @@
...
+| Account Type                   | The type of DV360 account used to sync. Either `Advertiser`, `Partner`, or `Publisher`. **Note:** Select `Publisher` only if you plan to connect to Google Ad Manager.    |
+| Advertiser ID                  | The ID of your DV360 Advertiser account. Can be found in your Google Account under Advertiser Settings → Basic Details → Advertiser ID.                  {:target="_blank"}.                                                                                                                     |
+
+Also toggle on the Send Track setting. 
+
+> info ""
+
+After you complete the set up process, allow up to 24 hours for Google to create the new audience list. Once the list is created, Segment can begin to sync users to that list. Google may require additional time to process the initial audience additions. The entire first sync to DV360 may require 24-48 hours to complete. As a result, the first few audience syncs after you create the audience may fail.
+
+Extra information from the API (settings, connection modes, etc.) are automatically pulled in here.
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-display-video-360/index.md:125]:
```suggestion
```

File: src/connections/destinations/catalog/actions-display-video-360/index.md [hunk 19]
```diff
@@ -0,0 +1,222 @@
...
+{% include content/sync-frequency-note.md %}
+
+## Migrate from Personas Google Display & Video 360 Destination 
+
+Segment will copy all of your existing Personas Display & Video 360 Destination configurations to Display and Video 360 (Actions). Once the migration is completed , you will be notified by email. 
+
+The sole mandatory action involves **re-authenticating** for each migrated Personans Display & Video 360 Destination. While you may already possess OAuth credentials, it is essential to undergo reauthentication. This step is necessary to seamlessly integrate new audiences or implement updates to existing ones.
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-display-video-360/index.md:132]:
```suggestion
The sole mandatory action involves **re-authenticating** for each migrated Personans Display & Video 360 destination. While you may already possess OAuth credentials, it is essential to undergo reauthentication. This step is necessary to seamlessly integrate new audiences or implement updates to existing ones.
```

File: src/connections/destinations/catalog/actions-display-video-360/index.md [hunk 20]
```diff
@@ -0,0 +1,222 @@
...
+beta: true
+https://github.com/segmentio/segment-docs/blob/d4623aadec6f118fb5aa7ca846a9e6214c0d1129/src/connections/destinations/catalog/personas-display-video-360/index.md
+---
+
+> info Display and Video 360 (Actions) operates using third-party cookies, its match rates are influenced by the extent to which these cookies are supported by browsers
+
+Google's [Display & Video (DV360)](https://marketingplatform.google.com/about/display-video-360/){:target="_blank"} service is an end-to-end campaign management tool that enables enterprise customers to plan, measure, and run display and video advertisements. 
+
+> info ""
+> **Note**: You can connect to a Google Ad Manager account. For more information, see [4. Create an audience and finish DV360 configuration](#4-create-an-audience-and-finish-dv360-configuration) below. When filling out the `User-Role Granted` selected `Publisher` if you place to connect to Google Ad Manager.
```
[the remark this answers is not in the document]
Comment (rchinn-segment) [reply src/connections/destinations/catalog/actions-display-video-360/index.md:15]: Wasn't quite sure what you meant here. Took a shot at editing, but let me know if we need to adjust it further.

File: src/connections/destinations/catalog/actions-display-video-360/index.md [hunk 21]
```diff
@@ -0,0 +1,222 @@
...
+
+> warning ""
+> **Important**: You must meet certain implementation criteria to use the DV360 integration:
+> - For web traffic, you must have a client-side `analytics.js` source.
+> - For mobile app traffic, you must have a mobile source.
+
+> info ""
+> **Note**: Since the release of `analytics-ios` version 4, Segment no longer collects IDFA automatically. To collect and pass IDFA to your DV360 integration, follow the steps for Ad Tracking and IDFA in the [Analytics-iOS mobile source](/docs/connections/sources/catalog/libraries/mobile/ios#ad-tracking-and-idfa) documentation.
+
+## Details
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-display-video-360/index.md:27]: Can you provide a bit more context to this section? Maybe just a sentence or two that explains what the fields and requirements listed below are used for?

**Review by rchinn-segment — CHANGES_REQUESTED** [review]
@maryamsharif thanks for submitting this! I left some comments and suggested edits to align this a bit more with our style guide.

File: src/connections/destinations/catalog/actions-display-video-360/index.md (hunk 20 above)
[the remark this answers is not in the document]
Comment (rvadera12) [reply src/connections/destinations/catalog/actions-display-video-360/index.md:16]: @rchinn-segment for more context a user can connect to Google Ads Manager using our DV360 destination, the only difference is that a user needs to fill out the user role granted setting as `Publisher` - open on ideas to how to clearly explain this to customers

File: src/connections/destinations/catalog/actions-display-video-360/index.md (hunk 21 above)
Comment (rvadera12) [reply src/connections/destinations/catalog/actions-display-video-360/index.md:28]: @rchinn-segment this is required to set up the whole destination, open to suggestion if we need to clarify further

File: src/connections/destinations/catalog/actions-display-video-360/index.md [hunk 22]
```diff
@@ -0,0 +1,222 @@
...
+
+> info ""
+> Since the release of `analytics-ios` version 4, Segment no longer collects IDFA automatically. To collect and pass IDFA to your DV360 integration, follow the steps for Ad Tracking and IDFA in the [Analytics-iOS mobile source](/docs/connections/sources/catalog/libraries/mobile/ios#ad-tracking-and-idfa) documentation.
+
+## Details
+
+> info ""
+> For users detected to originate from US states with privacy restrictions, using a Google User ID to populate user lists is deprecated, and will be eventually sunset. It's recommended that bidders populate user lists with their hosted match data for these users.
+
+- **Audience appears as**: An audience list with the name of your Engage Audience on the **DV360 All Audiences** screen
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-display-video-360/index.md:33]:
```suggestion
Keep the following settings and requirements in mind as you set up your DV360 (Actions) destination.

- **Audience appears as**: An audience list with the name of your Engage Audience on the **DV360 All Audiences** screen
```

File: src/connections/destinations/catalog/actions-display-video-360/index.md [hunk 23]
```diff
@@ -0,0 +1,224 @@
...
+
+> info ""
+> Since the release of `analytics-ios` version 4, Segment no longer collects IDFA automatically. To collect and pass IDFA to your DV360 integration, follow the steps for Ad Tracking and IDFA in the [Analytics-iOS mobile source](/docs/connections/sources/catalog/libraries/mobile/ios#ad-tracking-and-idfa) documentation.
+
+## Details
+
+> info ""
+> For users detected to originate from US states with privacy restrictions, using a Google User ID to populate user lists is deprecated, and will be eventually sunset. It's recommended that bidders populate user lists with their hosted match data for these users.
+
+Keep the following settings and requirements in mind as you set up your DV360 (Actions) destination.
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-display-video-360/index.md:33]:
```suggestion
Keep the following settings and requirements in mind as you set up your DV360 (Actions) Destination.
```

File: src/connections/destinations/catalog/actions-display-video-360/index.md [hunk 24]
```diff
@@ -0,0 +1,224 @@
...
+### DoubleClick Floodlight tag
+
+Segment users must add this tag to their web properties. The tag performs several functions, but when enabled for the DV360 integration, it allows Segment to send information about users directly to Google client-side.
+
+> info ""
+> This component is required only if you want to sync audiences based on web traffic.
+
+### DV360 destination
+
+The DV360 destination syncs audience data between Segment and Google Display & Video 360. For more information about enabling the DV360 destination, [view the setup instructions below](#set-up) below.
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-display-video-360/index.md:68]:
```suggestion
The DV360 Destination syncs audience data between Segment and Google Display & Video 360. For more information about enabling the DV360 Destination, [view the setup instructions below](#set-up) below.
```

File: src/connections/destinations/catalog/actions-display-video-360/index.md [hunk 25]
```diff
@@ -0,0 +1,224 @@
...
+
+## Setup
+
+Configuring this integration requires action by both you in your Segment workspace, and Google in your Google Marketing Platform account. As a result, the time required to finish configuration and setup can vary.
+
+### Configure client integration for web traffic
+
+> info ""
+> This step is necessary only if you want to use Google User IDs to build audiences based on website traffic. If you plan to use mobile identifiers only, continue to [Enable and configure the DV360 destination](enable-and-configure-the-dv360-destination).
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-display-video-360/index.md:77]:
```suggestion
> This step is necessary only if you want to use Google User IDs to build audiences based on website traffic. If you plan to use mobile identifiers only, continue to [Enable and configure the DV360 Destination](enable-and-configure-the-dv360-destination).
```

File: src/connections/destinations/catalog/actions-display-video-360/index.md [hunk 26]
```diff
@@ -0,0 +1,224 @@
...
+Google Chrome has committed to replacing third-party cookies with an alternative, but has not announced a timeframe for that alternative. Segment will not update this integration until these updates from Google are announced.
+
+
+### Can I use Engage audiences to target YouTube ads with this integration?
+
+No. YouTube (through DV360) does not support the type of lists that Segment provides.
+
+### Why do I see destination settings after I add my audience, but not when I first enable the destination?
+
+The DV360 destination works on a per-audience basis. This enables you to:
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-display-video-360/index.md:221]:
```suggestion
The DV360 Destination works on a per-audience basis. This enables you to:
```

File: src/connections/destinations/catalog/actions-display-video-360/index.md [hunk 27]
```diff
@@ -0,0 +1,224 @@
...
+5. Click **Save** and make sure to enable the mapping. 
+6. On the **Mappings** tab, click **Add Mapping** and select **Remove from Audience**. 
+7. Click **Save** and make sure you enable the mapping. 
+
+> info ""
+> The destination does not have configurable settings until you create an audience, described [here](#create-an-audience-and-finish-dv360-configuration).
+
+### Create an audience and finish DV360 configuration
+
+[Create an audience](/docs/personas/audiences) in a new or existing Engage space. After you create the audience, you can select the Display & Video 360 (Actions) destination you created before.
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-display-video-360/index.md:111]:
```suggestion
[Create an audience](/docs/personas/audiences) in a new or existing Engage space. After you create the audience, you can select the Display & Video 360 (Actions) Destination you created before.
```

File: src/connections/destinations/catalog/actions-display-video-360/index.md [hunk 28]
```diff
@@ -0,0 +1,224 @@
...
+
+Segment requires the [DoubleClick Floodlight](/docs/connections/destinations/catalog/doubleclick-floodlight/) tag on your website to enable the creation of audiences based on website traffic. This allows Segment to send Google the appropriate identifier (typically `anonymousId`) for users that are in an audience. Google stores these identifiers on its servers and matches them against `google_id`.
+
+To configure DoubleClick Floodlight:
+
+> warning ""
+> **Prerequisite**: Create a [JavaScript Website](/docs/connections/sources/catalog/libraries/website/javascript/) source in your Segment workspace if one does not exist. Ensure that this source is configured to track visitors to your website. For more information about configuring Javascript sources, see the [Analytics.js Quickstart guide](/docs/connections/sources/catalog/libraries/website/javascript/quickstart/).
+
+1. In your workspace, visit the **Catalog** and search for the **DoubleClick Floodlight** destination.
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-display-video-360/index.md:86]:
```suggestion
1. In your workspace, visit the **Catalog** and search for the **DoubleClick Floodlight** Destination.
```

File: src/connections/destinations/catalog/actions-display-video-360/index.md [hunk 29]
```diff
@@ -0,0 +1,224 @@
...
+2. Connect your JavaScript website source to the DoubleClick Floodlight destination, and configure the following settings:
+   1. **Get DoubleClickID**: `On`
+   2. **Google Network Id**: `segment`
+   3. Your [Segment Write Key](/docs/connections/find-writekey/). You can retrieve your write key from the Settings tab in the source.
+   4. **DoubleClick Advertiser ID**
+      - If you use DoubleClick Floodlight for DV360 only, enter `DV360`.
+      - If you use DoubleClick Floodlight for other use cases in addition to DV360, enter the Advertiser ID from your Doubleclick Floodlight account.
+3.  Switch the toggle to enable the destination.
+
+### Enable and configure the DV360 destination
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-display-video-360/index.md:96]:
```suggestion
### Enable and configure the DV360 Destination
```

File: src/connections/destinations/catalog/actions-display-video-360/index.md [hunk 30]
```diff
@@ -0,0 +1,224 @@
...
+
+After you complete the set up process, allow up to 24 hours for Google to create the new audience list. Once the list is created, Segment can begin to sync users to that list. Google may require additional time to process the initial audience additions. The entire first sync to DV360 may require 24-48 hours to complete. As a result, the first few audience syncs after you create the audience may fail.
+
+{% include content/sync-frequency-note.md %}
+
+## Migrate from Personas Google Display & Video 360 Destination 
+
+Segment will copy all of your existing Personas Display & Video 360 Destination configurations to Display and Video 360 (Actions). Once the migration is completed , you will be notified by email. 
+
+The sole mandatory action involves **re-authenticating** for each migrated Personans Display & Video 360 destination. While you may already possess OAuth credentials, it is essential to undergo reauthentication. This step is necessary to seamlessly integrate new audiences or implement updates to existing ones.
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-display-video-360/index.md:134]:
```suggestion
The sole mandatory action involves **re-authenticating** for each migrated Personans Display & Video 360 Destination. While you may already possess OAuth credentials, it is essential to undergo reauthentication. This step is necessary to seamlessly integrate new audiences or implement updates to existing ones.
```

File: src/connections/destinations/catalog/actions-display-video-360/index.md [hunk 31]
```diff
@@ -0,0 +1,224 @@
...
+
+Engage syncs every IDFA or `anonymous_id` value for each user in an audience. When DV360 receives this data, it does not de-duplicate in the event that multiple identifiers map to the same unique user. This may result in a larger audience list in Google compared to Engage.
+
+
+### Why don't I see matches in DV360?
+
+The most common cause of matches not appearing in DV360 is an error with Doubleclick Floodlight. From the website where tracking is enabled, open the Network inspector, and confirm that outgoing requests to `idsync.segment.com` appear.
+
+
+### How does third-party cookie eradication impact the DV360 destination?
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-display-video-360/index.md:210]:
```suggestion
### How does third-party cookie eradication impact the DV360 Destination?
```

File: src/connections/destinations/catalog/actions-display-video-360/index.md [hunk 32]
```diff
@@ -0,0 +1,224 @@
...
+Segment users must add this tag to their web properties. The tag performs several functions, but when enabled for the DV360 integration, it allows Segment to send information about users directly to Google client-side.
+
+> info ""
+> This component is required only if you want to sync audiences based on web traffic.
+
+### DV360 destination
+
+The DV360 Destination syncs audience data between Segment and Google Display & Video 360. For more information about enabling the DV360 Destination, [view the setup instructions below](#set-up) below.
+
+## Setup
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-display-video-360/index.md:70]:
```suggestion
## Set up the DV360 Destination
```

File: src/connections/destinations/catalog/actions-display-video-360/index.md [hunk 33]
```diff
@@ -0,0 +1,224 @@
...
+To configure DoubleClick Floodlight:
+
+> warning ""
+> **Prerequisite**: Create a [JavaScript Website](/docs/connections/sources/catalog/libraries/website/javascript/) source in your Segment workspace if one does not exist. Ensure that this source is configured to track visitors to your website. For more information about configuring Javascript sources, see the [Analytics.js Quickstart guide](/docs/connections/sources/catalog/libraries/website/javascript/quickstart/).
+
+1. In your workspace, visit the **Catalog** and search for the **DoubleClick Floodlight** Destination.
+2. Connect your JavaScript website source to the DoubleClick Floodlight destination, and configure the following settings:
+   1. **Get DoubleClickID**: `On`
+   2. **Google Network Id**: `segment`
+   3. Your [Segment Write Key](/docs/connections/find-writekey/). You can retrieve your write key from the Settings tab in the source.
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-display-video-360/index.md:90]:
```suggestion
   3. Your [Segment write key](/docs/connections/find-writekey/). You can retrieve your write key from the Settings tab in the source.
```

**Review by rchinn-segment — APPROVED** [review]

**Pull request merged**